pom.xml

```diff
@@ -930,7 +930,10 @@
               <goal>run</goal>
             </goals>
             <configuration>
-              <streamLogs>true</streamLogs>
+              <writeJunitReport>true</writeJunitReport>
+              <junitPackageName>io.jenkins.plugins.pom.its</junitPackageName>
+<!--              <streamLogs>true</streamLogs>-->
+              <streamLogsOnFailures>true</streamLogsOnFailures>
```

Comment:
just to have the failure directly in the build output in case of IT failure which may be a bit more convenient when running test locally. 
see output here https://ci.jenkins.io/job/Tools/job/plugin-pom/job/PR-508/11/console

Reply:
> when running test locally

OK. Or you can use the profile trick I developed in https://github.com/jenkinsci/maven-hpi-plugin/pull/301.